tianshou/data/batch.py

```diff
@@ -511,13 +531,35 @@ def stack_(self,
 
     @staticmethod
     def stack(batches: List['Batch'], axis: int = 0) -> 'Batch':
-        """Stack a :class:`~tianshou.data.Batch` object into a
-        single new batch.
+        """Stack a :class:`~tianshou.data.Batch` object into a single new
+        batch.
         """
         batch = Batch()
         batch.stack_(batches, axis)
         return batch
 
+    def empty_(self) -> None:
```

Comment:
It should return the updated Batch

Reply:
I think Batch.empty is doing this thing instead of Batch.empty_

Reply:
No, both should. See for instance https://pytorch.org/docs/stable/tensors.html#torch.Tensor.zero_